mscore/inspector/inspectorMeasureNumber.cpp

```diff
@@ -23,14 +22,17 @@ namespace Ms {
 InspectorMeasureNumber::InspectorMeasureNumber(QWidget* parent)
    : InspectorTextBase(parent)
       {
+      mn.setupUi(addWidget());
+
       const std::vector<InspectorItem> iiList = {
+            { Pid::SUB_STYLE, 0, mn.style, mn.resetStyle },
```

Comment:
Why isn't `Pid::PLACEMENT` present?

Reply:
Because the placement is related to the whole score.

Reply:
At first I added it too, but when toggling it the measure number switched from over the staff to below the staff. It wouldn't come under the whole system. So i thought I'd first have a working PR. And the inspector toggle would come after.
I also couldn't find in which cases it would be useful anyway.
But don't worry: I'm  working on that next PR ;)

Reply:
But I cannot tell whether it's working, I cannot find a way to flip it at all!